### ISSUE TITLE
Is that possible to render the output waveform of simulation on jupyter notebooks?

### ISSUE THREAD

**Issue by Readon** [jijingg/Spinal-bootcamp#12]
It would be useful to draw the simulated waveform on jupyter and analysis the result.
The easiest way is to enable the capability of render vcd file on jupyter.
Might the wavedrom widget helps?

**Comment by jijingg** [comment]
No, not now .  show a simple demo wave on  browser is ok. but for real IC design project trace wave on browser shold be too hard  . on the other hand wavedrom cant zoom， and there are no mature alternatives available .so right now for developer the common EDA tools（gtkwave , verdi ...)  is recommonded . 
 
In the future, there may be open source and easy-to-use debugging tools, we will wait and see

**Comment by jijingg** [comment]
@Readon No, not now . show a simple demo wave on browser is ok. but for real IC design project trace wave on browser shold be too hard . on the other hand wavedrom cant zoom， and there are no mature alternatives available .so right now for developer the common EDA tools（gtkwave , verdi ...) is recommonded .

In the future, there may be open source and easy-to-use debugging tools, we will wait and see

**Comment by Readon** [comment]
How about [fliplot](https://github.com/raczben/fliplot) ?
